diesel/src/pg/mod.rs

```diff
@@ -17,6 +17,7 @@ mod transaction;
 
 pub use self::backend::{Pg, PgTypeMetadata};
 pub use self::connection::PgConnection;
+pub use self::metadata_lookup::PgMetadataCache;
```

Comment:
That should not be public, it's an implementation detail.
```suggestion
```

Reply:
Should be good now :)